include/grpc/grpc_security.h

```diff
@@ -320,6 +347,31 @@ GRPCAPI grpc_server_credentials *grpc_ssl_server_credentials_create_ex(
     grpc_ssl_client_certificate_request_type client_certificate_request,
     void *reserved);
 
+typedef struct grpc_ssl_server_credentials_create_options
+    grpc_ssl_server_credentials_create_options;
+
+/** Creates an options object using a certificate store. Use this method when
+ * the certificates and keys of the SSL server will not changes during the
+ * server's lifetime. */
+GRPCAPI grpc_ssl_server_credentials_create_options *
+grpc_ssl_server_credentials_create_options_using_store(
+    grpc_ssl_client_certificate_request_type client_certificate_request,
+    grpc_ssl_server_certificate_store *certificate_store);
+
+/** Creates an options object using a certificate store callback. Use this
+ * method to reload the certificates and keys of the SSL server without
+ * interrupting the operation of the server. */
```

Comment:
if the user opts for the "callback" approach, then when they create the server, the core lib _will_ call their callback as part of initializing the server. i think it might be useful to mention this?